### DISCUSSION TITLE
[Feature] I stopped the sponsorship and here's why (related to missing features)

### DISCUSSION THREAD

**Discussion by Majestic7979** [immich-app/immich#7345] [Feature Request]
### The feature

Firstly, I was giving monthly. I stopped. While I am grateful that this project exists and is actively developed, I feel that I am not really able to use it effectively to support financially.

Here's why I stopped my github sponsorship:

- I reported a simple issue. It was turned into a feature request. It was not developed, despite being made several releases in the past (specifically, this one: https://github.com/immich-app/immich/discussions/3507)
- This application is still available only as a stack. I don't use compose. I want a monolithic container from the original source. Unfortunately I have to keep using "ghcr.io/imagegenius/immich:latest". While it works, it's basically causing duplication of labor, in addition to introducing complications and delays in updates that imagegenius has to deal with.
- Several users have requested Immich to expose a WebDAV share so assets can easily be accessed from file explorer. Not done. It's a pity because this would greatly simplify matters, since you could essentially have a repository of images and videos from any device and it would all be in sync. You could just drop files into it, and they'd get processed. While this is a more complex thing to develop, the fact that my first point (drag image from browser into desktop/folder) has not even been considered leads me to think that feature requests are not prioritized.
- Every time I want to delete a photo on the Android app, it asks me to confirm/allow. It sucks. There's a Trash for a reason, if I press delete just send to trash and delete 30 days later or whatever the limit is set by the user. I don't want to confirm every single time. Makes managing the library tedious. This would be greatly simplified with the introduction of WebDAV where I could quickly manage files from File Explorer.

Essentially, when I feel like I can pull a complete image with its own database and not have to worry, and also see reasonably easy feature requests implemented timely, then I'll reconsider. Immich is amazing for what it is but it's not yet there for me as a replacement to Gphotos. I hope it will be.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Thankfully, I didn't get triggered by this and stop my donation of my free time every day to Immich

**Comment by Majestic7979** [comment]
What a pity that you actually chose to respond that way to genuine feedback. Hubris is not a good human trait, Alex.

**Reply by bo0tzz** [reply]
I want to give a more elaborate response to hopefully give an insight into why Alex replied the way he did:

We really appreciate any support given, and of course you are free to stop your donations at any time you want. The way you stated your reasons for stopping support here came across as if you are unhappy because some features you want did not get implemented as fast as you would like, despite your donations. However, supporting Immich with donations does not have any bearing on priority of features, but is just what it says on the tin - a generous gift. Likewise, Alex (and the rest of the contributors) generously donate their time to work on Immich. At the moment there is so much work still to be done on Immich that every contributor is just working off their own priority list. In the future we will probably get to a point where we prioritize based on the feature requests in github, and then we might choose to consider donations in that, but at the moment that is not the case.

**Reply by alextran1502** [reply]
I agree it was not nice of me. My apologies

**Reply by Majestic7979** [reply]
"The way you stated your reasons for stopping support here **came across**..."

I had a whole comment typed but you know what? All I've got to say is you're wrong. It came across? That's the failure of the reader to understand. First paragraph I say the word "grateful". Last paragraph I end with the word "amazing", so how I "came across" as anything but nice is not my problem.

I appreciate that Alex has apologized, at least. We don't have to lose our decency and dignity. I never demanded anything. I said the app doesn't work for me for those reasons and hoped one day I could use it again if/when it was addressed. No demands there. In the meantime my contribution stopped because I am no longer using Immich, and provided feedback as to why.

I didn't deserve that treatment and the fact that several people reacted to your post as if you're right disgusts me, frankly.

No, it didn't "come across" - it's just that toxic people choose to read things badly. If words like amazing and grateful "come across" badly, clearly those people have issues. You stay well, I'm unsubscribing, I don't want this negativity in my life.

**Reply by mertalev** [reply]
To me, the phrasing "despite being made several releases in the past" implies that you expected your requests to be prioritized as a sponsor, or that your support was, in part, conditional on the implementation of a given feature.

Open source projects like these don't have a concept of deadlines, and we don't consider sponsorship as a basis for approving and prioritizing certain features.

You're of course free to decide for yourself the terms in which you'll sponsor a project, but this post places more importance on your individual sponsorship in relation to these features than is perhaps due.

While Immich is a fast-moving project with passionate developers, there are simply too many requests and too much work to be done for us to make any promises about which features will be implemented or the order in which this will be done. I apologize for any miscommunication that led you to believe otherwise.
